packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.routing.js

```diff
@@ -1,14 +1,19 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('network-security.traffic', {
-    url: '/traffic',
-    component: 'traffic',
-    params: {
-      subnet: null,
-    },
-    resolve: {
-      breadcrumb: /* @ngInject */ ($translate) =>
-        $translate.instant('network_security_dashboard_breadcrumb'),
-      getSubnet: /* @ngInject */ ($state) => () => $state.params.subnet,
-    },
-  });
+  $stateProvider
+    .state('network-security.traffic', {
+      url: '/traffic',
+      component: 'traffic',
+      params: {
+        subnet: null,
+      },
+      resolve: {
+        breadcrumb: /* @ngInject */ ($translate) =>
+          $translate.instant('network_security_dashboard_breadcrumb'),
+        getSubnet: /* @ngInject */ ($state) => () => $state.params.subnet,
+      },
+    })
+    .state('network-security.traffic.ip', {
+      url: '?ip&dateTime',
+      component: 'traffic',
+    });
```

Comment:
Why 2 states for same component ?
And in you component file you bind ip and dateTime that not resolve in routing but directly in you controller. I think we can simplify.

Reply:
I'm ok to simplify but all I'm trying doesn't work but the way I took worked. Detailed advice are welcome to make a simple solution.